pyproject.toml

```diff
@@ -178,21 +178,16 @@ repair-wheel-command = [
 ]
 
 [tool.cibuildwheel.windows]
-# This does not work, use CIBW_ENVIRONMENT_WINDOWS
-environment = {PKG_CONFIG_PATH="./.openblas"}
 config-settings = "setup-args=--vsenv setup-args=-Dallow-noblas=false build-dir=build"
 repair-wheel-command = "bash -el ./tools/wheels/repair_windows.sh {wheel} {dest_dir}"
+# This does not work, use CIBW_ENVIRONMENT_WINDOWS
+environment = {PKG_CONFIG_PATH="./.openblas"}
```

Comment:
Nit: can you undo this moving around? Nothing changes here, better to keep git blame clean

Reply:
Since I am touching pyproject.toml anyway, I would prefer to refactor this section, it reads much better this way. Before this change it was unclear what "This" referred to